index.html

```diff
@@ -54,9 +54,9 @@
     </header>
     <section class="main_section">
     <div class="main_section_div">
-      <h1 class="h1_title">Hey There, I’m Rapheal!</h1>
+      <h1 class="h1_title">Hey There, <br> I’m Rapheal!</h1>
```

Comment:
- [ ] Please note that when I open your project in a 400px screen it shows like this 🔽 , make sure to detect which of the recent changes caused your project to start behaving this way and fix it.

 
![2023-04-01_000821](https://user-images.githubusercontent.com/104319462/229255065-324fcb82-177d-4078-a92e-866f74fde7b7.png)

- [ ] And this is how it looks when I open it on an 800px width screen 🔽 note the white space in the right side, please fx this.

![2023-04-01_001119](https://user-images.githubusercontent.com/104319462/229255205-d95f754c-e563-4702-8ccb-9cddf2f62a3e.png)

Reply:
- [ ] As I stated in the first review and emphasized by the second reviewer this problem is still present in your project. please fix it.

> * [ ]  Please note that when I open your project in a 400px screen it shows like this 🔽 , make sure to detect which of the recent changes caused your project to start behaving this way and fix it.
> 
> ![2023-04-01_000821](https://user-images.githubusercontent.com/104319462/229255065-324fcb82-177d-4078-a92e-866f74fde7b7.png)
> 
> * [ ]  And this is how it looks when I open it on an 800px width screen 🔽 note the white space in the right side, please fx this.
> 
> ![2023-04-01_001119](https://user-images.githubusercontent.com/104319462/229255205-d95f754c-e563-4702-8ccb-9cddf2f62a3e.png)